rust/hyperlane-base/src/settings/trace/mod.rs

```diff
@@ -63,7 +63,7 @@ impl TracingConfig {
     pub fn start_tracing(&self, metrics: &CoreMetrics) -> Result<console_subscriber::Server> {
         let mut target_layer = Targets::new().with_default(self.level);
 
-        if self.level < Level::DependencyTrace {
+        if self.level == Level::Trace {
```

Comment:
not sure if it's bc of this change or not but seems like we now (or still ?) get some noisy logs

from e2e:
![Screen Shot 2024-06-26 at 2 07 15 PM](https://github.com/hyperlane-xyz/hyperlane-monorepo/assets/20362969/d8aba85e-8062-4dc0-98ea-509b911354f4)

Wonder if we should set hyper to Level::Warn in the `if self.level < Level::Trace` block on line 82 or something - or maybe all of these?